### PR TITLE
example of a change

### PR DESCRIPTION
**Pull request by a-m-brewer**

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a file
```
Comment (Farnie) [inline file.txt:1]: I'm not convinced that this is a file, can you prove it?

**Review by Farnie — CHANGES_REQUESTED** [review]
So, although I left my comment against that file, I also need to submit this! :')

File: file.txt (hunk 1 above)
Comment (a-m-brewer) [reply file.txt:1]: Should I add a test for this text change?

Comment (Farnie) [reply file.txt:1]: No, move on with your life :')

**Pull request merged**